packages/canon/src/components/Button/styles.css

```diff
@@ -14,69 +14,70 @@
  * limitations under the License.
  */
 
-.button {
+.cn-button {
   all: unset;
   display: inline-flex;
   align-items: center;
   justify-content: center;
   user-select: none;
   font-family: var(--canon-font-regular);
-  font-weight: var(--canon-font-bold);
-  font-size: var(--canon-font-size-md);
+  font-weight: var(--canon-font-weight-bold);
+  font-size: var(--canon-font-size-body);
   padding: 0;
   transition: all 150ms ease;
   cursor: pointer;
   border-radius: 8px;
-}
 
-.button.primary {
-  background-color: var(--canon-accent);
-  color: var(--canon-text-primary-on-accent);
-}
+  &.cn-button-primary {
+    background-color: var(--canon-accent);
+    color: var(--canon-text-primary-on-accent);
 
-.button.primary:hover {
-  background-color: transparent;
-  box-shadow: inset 0 0 0 1px var(--canon-outline-focus);
-  color: var(--canon-text-primary);
-}
+    &:hover {
+      background-color: transparent;
+      box-shadow: inset 0 0 0 1px var(--canon-outline-focus);
+      color: var(--canon-text-primary);
+    }
+  }
 
-.button.secondary {
-  background-color: transparent;
-  box-shadow: inset 0 0 0 1px var(--canon-outline);
-  color: var(--canon-text-primary);
-}
+  &.cn-button-secondary {
+    background-color: transparent;
+    box-shadow: inset 0 0 0 1px var(--canon-outline);
+    color: var(--canon-text-primary);
 
-.button.secondary:hover {
-  box-shadow: inset 0 0 0 1px var(--canon-outline-hover);
-}
+    &:hover {
+      box-shadow: inset 0 0 0 1px var(--canon-outline-hover);
+    }
+  }
 
-.button.tertiary {
-  background-color: transparent;
-  color: var(--canon-text-primary);
-}
+  &.cn-button-tertiary {
+    background-color: transparent;
+    color: var(--canon-text-primary);
 
-.button.tertiary:hover {
-  color: var(--canon-text-secondary);
-}
+    &:hover {
+      color: var(--canon-text-secondary);
+    }
+  }
 
-.button.small {
-  padding-left: 6px;
-  padding-right: 6px;
-  height: 32px;
-}
+  &.cn-button-small {
+    padding-left: 6px;
+    padding-right: 6px;
+    height: 32px;
+  }
 
-.button.medium {
-  padding-left: 8px;
-  padding-right: 8px;
-  height: 40px;
-}
+  &.cn-button-medium {
+    padding-left: 8px;
+    padding-right: 8px;
+    height: 40px;
+  }
 
-.button-content {
-  display: flex;
-  align-items: center;
-  gap: var(--canon-spacing-xs);
-}
+  & .cn-button-content {
```

Comment:
Thinking it may be best to have this be standalone so that it's possible to override with a simple `.cn-button-content { ... }`  rather than needing to include `.cn-button` too. In general I'd say it's very important to form some strict rules around how specific we get with selectors, in order to make it simple to override styles if we go with the CSS approach

Reply:
Oh you're right. Making sure these styles are flat. I'll do the update.

Reply:
Ok this is fixed now.